modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/util/TaxonomyCategoryBriefUtil.java

```diff
@@ -48,6 +52,33 @@ public static TaxonomyCategoryBrief toTaxonomyCategoryBrief(
 		};
 	}
 
+	private static Scope _getScope(AssetCategory assetCategory) {
+		Group group = GroupLocalServiceUtil.fetchGroup(
+			assetCategory.getGroupId());
+
+		if (group == null) {
+			return null;
+		}
+
+		return new Scope() {
```

Comment:
Here, you are creating a Scope from a Group. I saw also [here](https://github.com/liferay/liferay-portal/blob/6994b14cc04de46280b2a72ac4670d7ca563ffdd/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java#L774) the same functionality. Should we move this to a common Util or DTOConverter class?